Makefile

```diff
@@ -29,14 +29,14 @@ BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
 #
 # For example, running 'make bundle-build bundle-push catalog-build catalog-push' will build and push both
 # redhat.com/operator-certification-operator-bundle:$VERSION and redhat.com/operator-certification-operator-catalog:$VERSION.
-IMAGE_TAG_BASE ?= redhat.com/operator-certification-operator
+IMAGE_TAG_BASE ?= quay.io/opdev/operator-certification-operator
 
 # BUNDLE_IMG defines the image:tag used for the bundle.
 # You can use it as an arg. (E.g make bundle-build BUNDLE_IMG=<some-registry>/<project-name-bundle>:<tag>)
-BUNDLE_IMG ?= $(IMAGE_TAG_BASE)-bundle:v$(VERSION)
+BUNDLE_IMG ?= $(IMAGE_TAG_BASE)-bundle:$(VERSION)
 
 # Image URL to use all building/pushing image targets
-IMG ?= controller:latest
+IMG ?= quay.io/opdev/operator-certification-operator:$(VERSION)
```

Comment:
Can we use IMAGE_TAG_BASE here? $(IMAGE_TAG_BASE):$(VERSION)

Reply:
We can.  but actually I would like to take another look at it because of $(VERSION).

VERSION is set to 0.0.1 at the top of the Makefile,  so installing the bundle currently won't work since there is no operator image tagged 0.0.1.  there's one with git commit.

Same applies to CSV.

Looking at ways to resolve this.  don't merge yet.

Reply:
/do-not-merge